reactor-core/src/test/java/reactor/core/publisher/FluxPublishOnTest.java

```diff
@@ -187,14 +181,16 @@ public static void after() {
 	public void failPrefetch() {
 		assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> {
 			Flux.range(1, 10)
-					.publishOn(Schedulers.immediate(), -1);
+			    .prefetch(-1)
```

Comment:
this one should be moved to `FluxPrefetchTest.java` (and shouldn't need the `publishOn`)